eahub/base/models.py

```diff
@@ -11,11 +14,29 @@ def has_profile(self):
 
 class FeedbackURLConfig(SingletonModel):
     site_url = models.TextField(
-        default="feedback.eahub.org", validators=[URLValidator()]
+        default="https://feedback.eahub.org", validators=[URLValidator()]
```

Comment:
changed default in migration file in previous MR, but forgot to update model. This sets this right